WordPress/WordPressApi/WordPressComApi.m

```diff
@@ -7,7 +7,7 @@
 #import "NotificationsManager.h"
 #import "WPUserAgent.h"
 
-static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/v1.1/";
+static NSString *const WordPressComApiClientEndpointURL = @"https://public-api.wordpress.com/rest/";
```

Comment:
I'm wondering what happens if one of the developers doesn't notice we've changed how we use `WordPressComApi` and calls, for instance, `GET:@"me"`. Something that used to work will return a 404. Perhaps we should add some assertions to the GET/POST/PUT/DELETE methods to check that the URL starts with `v%d`